tests/test_archanaskitchen.py

```diff
@@ -1,3 +1,4 @@
+# canonical_url : https://www.archanaskitchen.com/classic-greek-salad-recipe
```

Comment:
Same note re: potential conversion into a skipped-test.

Reply:
(in fact, I see this pattern appears in a bunch of places - I don't think I'll comment on them all individually, but if you agree with the skippable-test approach, then lets migrate them all)